h2/src/test/org/h2/samples/FunctionMultiReturn.java

```diff
@@ -68,7 +69,7 @@ public static void main(String... args) throws Exception {
             double r = rs.getDouble(1);
             double a = rs.getDouble(2);
             Object o = rs.getObject(3);
-            Object[] xy = (Object[]) o;
+            Object[] xy = (Object[])((JdbcArray) o).getArray();
```

Comment:
`JdbcArray` is an internal class, it shouldn't be used here.

`java.sql.Array` or `Double[]` should be used instead.

```Java
Array o = rs.getArray(3);
// or
Double[] xy = rs.getObject(3, Double[].class);
```

Reply:
Hi @katzyn - I've updated the code based on your recommendations.  Please let me know if you have any other recommendations.   Thanks for your time.